Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -84,6 +84,16 @@ TEST_CASE(test_bmp_too_many_palette_colors)
     TRY_OR_FAIL(expect_single_frame_of_size(*plugin_decoder, { 191, 231 }));
 }
 
+TEST_CASE(test_bmp_v4)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("bmp/400x300x32_v4.bmp"sv)));
```

Comment:
The BMPs you added to this PR are relatively large (hundreds of KiB) but I suspect they don't need to be to confirm you've fixed these bugs - can you use smaller images?

Reply:
Sure, managed to re-create these files, but in much smaller versions. I'm also not sure how many tests do we want for image decoders. I decided to add one per each fix just in case, but please let me know if we don't need that many test caes in the future :^)

Reply:
Adding these tests is absolutely fine, the more the merrier!